Simplified/AccountsManager.swift

```diff
@@ -19,10 +19,10 @@ let prodUrlHash = prodUrl.absoluteString.md5().base64EncodedStringUrlSafe().trim
  cacheOnly - only fetch from cache, unless `noCache` is specified
  @param completion callback method when this is complete, providing the data or nil if unsuccessful
  */
-func loadDataWithCache(url: URL, cacheUrl: URL, options: AccountsManager.LoadOptions, completion: @escaping (Data?) -> ()) {
+func loadDataWithCache(url: URL, cacheUrl: URL, expiryComponent: Calendar.Component = .day, expiryValue: Int = 1, options: AccountsManager.LoadOptions, completion: @escaping (Data?) -> ()) {
```

Comment:
expiryComponent is kind of unclear. Maybe stick with the same terminology apple uses in `Calendar::date(byAdding:value:to:options:)`, so `expiryUnit`.

Reply:
Updated the param name to `expiryUnit` and removed `preferringCache` param from `loadAuthenticationDocument`